### PR TITLE
Before running functional tests on CI, do a VM test deployment

### PR DESCRIPTION
**Pull request by rmohr**
To make sure, that all services are really ready, before running the
tests, do a VM test deployment.

### CODE REVIEW

File: tests/spice_test.go [hunk 1]
```diff
@@ -63,7 +63,7 @@ var _ = Describe("Vmlifecycle", func() {
 			Expect(section.Key("port").Value()).To(Equal("4000"))
 			Expect(section.HasKey("proxy")).To(BeTrue())
 			close(done)
-		}, 30)
+		}, 50)
```
Comment (admiyo) [inline tests/spice_test.go]: make this a constant

File: tests/utils.go [hunk 2]
```diff
@@ -250,11 +251,19 @@ func WaitForSuccessfulVMStart(vm runtime.Object) (nodeName string) {
...
+				return ""
+			} else {
+				Expect(err).ToNot(HaveOccurred())
+			}
+		}
 		fetchedVM := obj.(*v1.VM)
 		nodeName = fetchedVM.Status.NodeName
 		return fetchedVM.Status.Phase
-	}).Should(Equal(v1.Running))
+	}, 10*time.Second).Should(Equal(v1.Running))
```
Comment (admiyo) [inline tests/utils.go]: Constant

File: tests/spice_test.go (hunk 1 above)
Comment (rmohr) [inline tests/spice_test.go]: I am not so sure where to use constants in the tests. Maybe one solution would be to define a standard test run time with a constant, and then multiply it, first with a test run factor  and then with a system speed factor?

**Comment by rmohr** [comment]
retest this please

**Comment by davidvossel** [comment]
retest this please

**Comment by rmohr** [comment]
This led to nowhere.

**Pull request closed without merging**